src/pages/yields/pool/[...pool].js

```diff
@@ -207,10 +199,15 @@ export async function getStaticProps({
 
 	const res = await arrayFetcher([poolUrl, chartUrl])
 
+	const poolData = res[0]?.data?.[0] ?? {}
+
+	const configData = await fetch(`${CONFIG_API}/smol/${poolData.project}`).then((res) => res.json())
```

Comment:
might make sense to fetch all resources in parallel to speed up the function

Reply:
but we need project name from that pool api response